### ISSUE TITLE
Use pretrained Model to provide reliable Embeddings

### ISSUE THREAD

**Issue by Jxshyz** [tis22/image_recommender#6]
pretrained model can be choosen freely on (https://pytorch.org/vision/0.17/models.html)

i suggest (https://pytorch.org/vision/0.17/models/generated/torchvision.models.efficientnet_v2_s.html#torchvision.models.EfficientNet_V2_S_Weights) since its rather light but has stunning accuracy with 84%

##### Check Liveocding 08 V3 for guidance
-> image must be transformed to fit pretrained model dimensions
-> model weights must be load and set to .eval()
-> data must be fitted: we need to extract the embeddings
